elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideStandaloneSettings.java

```diff
@@ -253,6 +254,24 @@ default AsyncQueryDAO getAsyncQueryDAO() {
         return null;
     }
 
+    /**
+     * Implementation of ResultStorageEngine to use.
+     *
+     * @return ResultStorageEngine type object.
+     */
+    default ResultStorageEngine getResultStorageEngine() {
```

Comment:
Maybe all of these individual async settings shoudl be consolidated into a configuration object that is returned by a single setting function.